graphql/schema.js

```diff
@@ -91,7 +91,7 @@ type Commission {
 }
 
 type Compensation {
-  year: Int
+  year: Int!
```

Comment:
Are you sure that all `Compensation` have a year? The API will throw an error if not and not deliver the (schema wise faulty) result.

Reply:
Yeah, it is mandatory